Automattic/wp-calypso
# Hosting Page Cards: make all icons 32px.

Related to #

## Proposed Changes

* A small PR to correct the inconsistent icons on the hosting page. Most icons are 32px but the bottom two were 24px. 

Before

<img width="725" alt="Screenshot 2566-02-27 at 15 10 35" src="https://user-images.githubusercontent.com/6851384/221509302-68f0671f-0ed2-4aa8-a66e-6d9be9e1d79b.png">

After

<img width="765" alt="Screenshot 2566-02-27 at 15 03 43" src="https://user-images.githubusercontent.com/6851384/221509375-0d8be7d4-3227-4e17-a1b8-c6d6938c4120.png">




## Testing Instructions



* Look at the current `/hosting-config` page in prod and confirm icons are of different sizes
* Check Calypso Live and see the problem is fixed

## Pre-merge Checklist
